tests/test-sources/plugins/pluginmanagers/lz-n.nix

```diff
@@ -0,0 +1,140 @@
+{ pkgs, helpers, ... }:
+let
+  inherit (pkgs) lib;
+  optionalPlugins = map (plugin: {
+    inherit plugin;
+    optional = true;
+  });
```

Comment:
This function would benefit from a comment regarding it's purpose: what it is for, why it is needed and (if not obvious) how it works.